src/containers/Topics.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react'
+import { TopicsComponents } from 'twreporter-react-listing-components'
+import twreporterRedux from 'twreporter-redux'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+import { date2yyyymmdd, formatPostLinkTo, formatPostLinkTarget } from '../utils'
+import { LINK_PREFIX } from '../constants/'
+import More from '../components/More'
+
+import map from 'lodash/map'
+import get from 'lodash/get'
+
+const _ = {
+  get,
+  map
+}
+
+const { actions, reduxStateFields, utils } = twreporterRedux
+const { fetchTopics, fetchAFullTopic } = actions
+const { denormalizeTopics } = utils
+const { PageContent, TopSection, ListSection, PostsContainer, PostItem, TopicItem } = TopicsComponents
+const N_OF_FIRSTPAGE = 5
+const N_PER_PAGE = 5
+
+const PageContainer = styled.div`
+  position: relative;
+  width: 100%;
+  padding: 30px 0 0 0;
+`
+
+class Topics extends Component {
```

Comment:
Some of props from redux are objects, will shallow comparing always return that the props are different?